dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/Zorder.scala

```diff
@@ -42,21 +43,48 @@ case class Zorder(children: Seq[Expression]) extends Expression with CodegenFall
   }
 
   @transient
-  private lazy val defaultNullValues: Seq[Any] = {
-    children.map(child => ZorderBytesUtils.defaultValue(child.dataType))
-  }
+  private lazy val defaultNullValues: Array[Array[Byte]] =
+    children.map(_.dataType)
+      .map(ZorderBytesUtils.defaultValue)
+      .map(ZorderBytesUtils.toByte)
```

Comment:
@cfmcgrady we can simplify `ZorderBytesUtils.defaultValue` to return `Array[Byte]` directly. So we don't need call .map twice.

Reply:
Good catch, will raise a new pr.